Don't use -march=native. On non-Intel machines it will cause a build
failure and on Intel machines it may cause the built binary not to be
usable on computers with processor models different from the build
machine.
--- CMakeLists.txt.orig	2020-04-28 04:40:33.000000000 -0500
+++ CMakeLists.txt	2022-01-26 01:32:18.000000000 -0600
@@ -15,7 +15,7 @@
 
 include_directories(fasttext)
 
-set(CMAKE_CXX_FLAGS " -pthread -std=c++11 -funroll-loops -O3 -march=native")
+set(CMAKE_CXX_FLAGS " -pthread -std=c++11 -funroll-loops -O3")
 
 set(HEADER_FILES
     src/args.h
--- setup.py.orig	2020-04-28 04:40:33.000000000 -0500
+++ setup.py	2022-01-26 01:33:01.000000000 -0600
@@ -75,8 +75,8 @@
             FASTTEXT_SRC,
         ],
         language='c++',
-        extra_compile_args=["-O0 -fno-inline -fprofile-arcs -pthread -march=native" if coverage else
-                            "-O3 -funroll-loops -pthread -march=native"],
+        extra_compile_args=["-O0 -fno-inline -fprofile-arcs -pthread" if coverage else
+                            "-O3 -funroll-loops -pthread"],
     ),
 ]
 
